lib/keyword_extractor.js

```diff
@@ -1,6 +1,6 @@
 var _ = require("underscore");
 _.str = require('underscore.string');
-var supported_languages = ["english","spanish","polish", "german", "french", "italian", "dutch", "russian"];
+var supported_languages = ["english","spanish","polish", "german", "french", "italian", "dutch", "russian", "portuguese", 'portugueseTwo'];
```

Comment:
@RodrigoBastos Why two separate Portuguese entries? I feel like there could be a clearer name than `PortugueseTwo`, if there's a need for two separate entries.